src/components/Button/Button.css

```
@@ -0,0 +1,103 @@
+*{
+    @import url('https://fonts.googleapis.com/css2?family=Mulish:wght@300;400;700&display=swap');
+}
+.App{
+    display: flex;
+    flex-direction: row;
+}
+.App button{
```

В файле стилей кнопки должны быть стили относящиеся только к кнопке.
Стили для App нужно перенести в стили компонента App.
Присваивай основной класс по названию компонента без использования тегов в файле стилей
```suggestion
.button {
```